src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side1;
+
+    public Square(String color, int side1) {
+        super(color);
+        this.side1 = side1;
```

Comment:
To improve code readability, avoid passing expressions directly in the constructor. Instead, create a separate variable for 'side1' before passing it to the constructor.